### PR TITLE
Initial import of files from libero/community

### PR DESCRIPTION
**Pull request by NuclearRedeye**
To get this all kicked off, I've started by importing the essential files from the Libero Community repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+3. Run all tests and ensure they pass to check your environment is working
+4. Make your change, adding new tests
+5. Run all tests and ensure they pass
+6. Push to your fork and make a pull request
+
+We try to respond to pull requests with an initial comment within 3 working days.
+
+You can help your pull request be accepted by:
+- Writing tests
+- Following the coding standards - (e.g. [PHP coding standards](https://github.com/libero/php-coding-standard))
```
Comment (giorgiosironi) [inline CONTRIBUTING.md]: needs updating - this file comes from the previous Publisher architecture

Comment (NuclearRedeye) [reply CONTRIBUTING.md]: I've updated to point at libero/eslint instead, not sure if this is right though. Alternative would be to remove the text and create an issue to get a proper standards in place and to update this guide when ready.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Contributing to Libero
+
+We encourage contributions of any kind. By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or make sure your employer has signed the CLA on your behalf. If you have questions about the CLA please email the team at libero@elifesciences.org.
+
+All Libero repositories will have further instructions in their own `CONTRIBUTING.md` file.
```
Comment (giorgiosironi) [inline CONTRIBUTING.md]: or they won't have one, if this is used as the default?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+We try to respond to pull requests with an initial comment within 3 working days.
+
+You can help your pull request be accepted by:
+
+- Writing tests
+- Following the coding standards - (e.g. [Libero coding standards](https://github.com/libero/eslint-config))
+- Writing [good commit messages](https://chris.beams.io/posts/git-commit/)
+
+To make a larger contribution, [create an issue on the Libero repo](https://github.com/libero/libero/issues/new/choose) or start a discussion on the [Libero Community Slack](https://libero-community.slack.com/). This helps to get early feedback on design and architecture.
```
Comment (giorgiosironi) [inline CONTRIBUTING.md]: the first link redirects to `libero/publisher`. Use `libero/community` instead?

Comment (NuclearRedeye) [reply CONTRIBUTING.md]: Interesting, did libero/publisher used to be libero/libero? Could use community, but is this where we want these larger contribution issues to be raised? @BlueReZZ

Comment (giorgiosironi) [reply CONTRIBUTING.md]: > did libero/publisher used to be libero/libero

This part: yes.

File: CONTRIBUTING.md (hunk 2 above)
Comment (NuclearRedeye) [reply CONTRIBUTING.md]: Ah yeah, that line 'should' be redundant once I merge this, hence best course of action is to remove it.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
```
Comment (diversemix) [inline CODE_OF_CONDUCT.md:37]: I'm not a fan of bland/vague statements like this ... I'd like to define "harassment" as under the Equalities Act 2010 as http://www.legislation.gov.uk/ukpga/2010/15/section/26

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+- Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
```
Comment (diversemix) [inline CODE_OF_CONDUCT.md:40]: Again this lacks details on how the term "inappropriate" is defined and who / how its going to be judged.

**Review by diversemix — COMMENTED** [review]
In general I'm "ok" adopting the CoC - but we should all read and be aware of the potential exposure this opens us up to. It is extremely vague and so is open to misuse, this should not pose an issue to us at the moment but I think we can do better! 

It seems a lot more inclusive to express values rather than trying to catalogue the potential bad behaviours (example: https://docs.fedoraproject.org/en-US/project/code-of-conduct/ ) - This to me seems a more sensible approach both socially and from a mathematical set theory approach. 

These sorts of issues have been discussed at great length in both the Ruby and OpenStreetMap communities, here is what I have read:
* https://bugs.ruby-lang.org/issues/12004
* https://github.com/openstreetmap/openstreetmap-website/issues/1299

**Comment by giorgiosironi** [comment]
Trying to sidestep the issue here: this is a PR on copying the code of conduct from one repository to another, not on the merits of code of conducts which is a much larger discussion.

**Comment by NuclearRedeye** [comment]
@giorgiosironi @diversemix I'd say lets get these committed as is, and then we can put some issues in to address your concerns.

**Comment by NuclearRedeye** [comment]
@giorgiosironi @diversemix @BlueReZZ Any further thoughts on this, would like to get it merged! Happy to address minor issues, but larger stuff should be raised as new issues.

**Review by giorgiosironi — APPROVED** [review]
Movement of files is taken care of here, not going into the content if not for checking there are no Publisher-specific references.

**Comment by BlueReZZ** [comment]
As @giorgiosironi mentions this is currently about moving the files so I think we should merge it as it is, given that not having it could leave some repos without a code of conduct - which I would consider to be a worse scenario.

We can raise the suggested changes as a pull request or a community RFC after that.

For context, the original code of conduct was reviewed and compared with other options by the current community at the time (Sept 2018) with a discussion on [the #governance channel in the Libero Community Slack](https://libero-community.slack.com/archives/CC7TPDLLC/p1536163332000100). People in that discussion now form part of the Steering Committee so perhaps this would be something to raise with them first.

**Review by BlueReZZ — APPROVED** [review]
Files look okay to merge. We can follow up with suggested changes to the code of conduct through the RFC or Steering Group process

**Pull request merged**